homeassistant/components/anthemav/__init__.py

```diff
@@ -41,6 +41,8 @@ def async_anthemav_update_callback(message):
 
     hass.config_entries.async_setup_platforms(entry, PLATFORMS)
 
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, avr.close)
```

Comment:
```suggestion
    entry.async_on_unload(
        hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, avr.close)
    )
```

Reply:
Should `avr.close` run in the event loop? If so we need to make an intermediate function decorated with `@callback` that calls `avr.close`.

Reply:
It's a good question, but I actually don't know. I'll change it just to be safe

Reply:
It should probably run in the event loop since we run it in the event loop when unloading the config entry and the library supports asyncio.